distribution/debian/install.sh

```diff
@@ -16,51 +17,136 @@
 #OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 #WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-scriptversion="1.0.3"
-scriptdate="2024-01-06"
+### Colors
+green='\033[0;32m'
+yellow='\033[1;33m'
+red='\033[0;31m'
+brown='\033[0;33m'
+reset='\033[0m' # No Color
 
-set -euo pipefail
+scriptversion="1.0.4"
+scriptdate="2024-04-10"
 
-echo "Running Sonarr Install Script - Version [$scriptversion] as of [$scriptdate]"
+set -euo pipefail
 
 # Am I root?, need root!
-
 if [ "$EUID" -ne 0 ]; then
-    echo "Please run as root."
+    echo -e ${red}"Please run as root!"
+    echo -e "Exiting script!"
     exit
 fi
 
+# Stuff the gremlins needs to know
 app="sonarr"
 app_port="8989"
 app_prereq="curl sqlite3 wget"
 app_umask="0002"
 branch="main"
 
-# Constants
+### CONSTANTS
```

Comment:
Why three `#`?
This appears to be the only comment that is all upper-case, `# Constants` is more consistent.